### PR TITLE
feat(ValidationMessage): icon when error

### PR DESCRIPTION
**Pull request by Barsnes**
this syncs code with figma

### CODE REVIEW

File: packages/css/validation-message.css [hunk 1]
```diff
@@ -17,5 +23,16 @@
...
+
+    &::before {
+      content: '';
+      display: inline-block;
+      height: var(--dsc-validation-message-icon-size);
+      width: var(--dsc-validation-message-icon-size);
+      margin-right: var(--dsc-validation-message-icon-gap);
+      vertical-align: top;
+      background-color: currentcolor;
+      mask: var(--dsc-validation-message-icon-url) center/contain no-repeat;
```
[the remark this answers is not in the document]
Comment (Barsnes) [reply packages/css/validation-message.css:35]: @Viveca24 should probably answer this

[the remark this answers is not in the document]
Comment (eirikbacker) [reply packages/css/validation-message.css]: Example: 
image
image

[the remark this answers is not in the document]
Comment (Viveca24) [reply packages/css/validation-message.css]: We should solve this like we did with Alert, using proper padding and positioning for a cleaner look without text wrapping around the icon.
Example: Padding and position
Example 2

Example: Text wrap around the icon
Example 1

**Review by mimarz — COMMENTED** [review]
Question in general, when we have no children, should the icon still show? 

image

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**